.github/dependabot.yml

```diff
@@ -39,6 +39,7 @@ updates:
     labels:
       - "type:dependencies"
       - "type:dependabot"
+      - "type:backport"

```

Comment:
@makubacki 

A backport label in a repo that doesn't have the dev/release split should not pose a problem, correct?

Reply:
It won't cause scripts or anything to act on the label, but it would be confusing to a user.

You'd need to update the synced template files anyway in [.sync/dependabot](https://github.com/microsoft/mu_devops/tree/main/.sync/dependabot). So, you could make those be Nunjucks templates and provide an optional parameter to add this label. And then break out the repos that have the dev/release branch split in `.sync/Files.yml` to include it during the sync.

Reply:
@apop5, are you planning to follow up with this change?

Reply:
Yes, just trying to finish up a couple of other things and will get back to this.